### ISSUE TITLE
Running make to build the exchange-api in a container works fine

### ISSUE THREAD

**Issue by codejaeger** [open-horizon/exchange-api#473]
Is the following section in the [README.md](https://github.com/open-horizon/exchange-api/blob/master/README.md) supposed to be updated?

> - **This step doesn't work yet, use the 2 steps in the following bullet for now
>    - To compile your local code, build the exchange container, and run it, run: `make`
> - Or you can build and run in separate steps:
>    -    Compile your local code and build the exchange container: make .docker-exec
>    -   Run the container locally: make start-docker-exec

The first option seems to work (at least to get the container running), the alternative does not.
Thanks.

**Comment by bmpotter** [comment]
@codejaeger yes it does need to be updated. Would you like to be assigned this issue? I'm going to do that now, let me know if you don't want it. The 2-step process needs to use target `.docker-exec-run` instead of `start-docker-exec`. After you get the docker container running, do a few quick tests by running a couple exchange APIs. Thx!

**Comment by bmpotter** [comment]
@codejaeger I updated the exchange `README.md` sections for starting postgres and for running the exchange in a local container (in PR [482](https://github.com/open-horizon/exchange-api/pull/482) ) and i think it all works now. So moving this to Review/QA
